app/mailers/report_mailer.rb

```diff
@@ -3,13 +3,19 @@ class ReportMailer < ApplicationMailer
   # TODO: change this for production
   default to: 'lihan@adhocteam.us'
 
-  def year_to_date_report_email(_report)
-    # TODO: add report as s3 upload
-    # attachments['report.csv'] = report
+  def year_to_date_report_email(report_file)
+    s3 = Aws::S3::Resource.new(
+      region: ENV['REPORTS_AWS_S3_REGION'],
+      access_key_id: ENV['REPORTS_AWS_ACCESS_KEY_ID'],
+      secret_access_key: ENV['REPORTS_AWS_SECRET_ACCESS_KEY']
+    )
+    obj = s3.bucket(ENV['REPORTS_AWS_S3_BUCKET']).object("#{SecureRandom.uuid}.csv")
```

Comment:
and the bucket in an `s3_bucket` method - both would read faster and make tracebacks when ENV vars aren't available a little easier to parse out